classes/autoptimizeCriticalCSSCore.php

```diff
@@ -203,7 +206,12 @@ public function ao_ccss_extend_types() {
         }
 
         // Templates.
-        $templates = wp_get_theme()->get_page_templates();
+        // Transient cache to avoid frequent disk reads
+        $templates = get_transient('ao_ccss_page_tpls');
```

Comment:
Minor: can you add spaces after `(` and before `)` in your additions (except when `()` )? :-)

Reply:
Sure, updated now as per existing coding style.